src/Common/src/System/Numerics/Hashing/HashHelpers.cs

```diff
@@ -6,12 +6,14 @@ namespace System.Numerics.Hashing
 {
     internal static class HashHelpers
     {
+        public static int RandomSeed { get; } = Guid.NewGuid().GetHashCode();
```

Comment:
Guid has enough randomness in it, even though it is not completely random. We use the same trick with Guid in `https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Random.cs#L154`.

Somebody raised concern in offline conversation that guid generation is somewhat expensive operation because of it fetches network card ID, etc. I am not sure how real the problem is. If it is, a good alternative may be "new Random().Next(Int32.MinValue, Int32.MaxValue)". It basically piggy backs on whatever is done to generate the random seed for `System.Random`.

`System.Random` also uses Guid right now but it is easy to fix if the guid generation is a concern. It can call "Microsoft.Win32.Random" that is a thin wrapper around `/dev/random` on Unix and BCryptGenRandom on Windows.